.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": " Codespace",
+	"image": "mcr.microsoft.com/vscode/devcontainers/dotnet:0.201.9-5.0",
```

Comment:
Can we use newer "mcr.microsoft.com/vscode/devcontainers/dotnet:3.1" image version?